scripts/lint.sh

```diff
@@ -88,13 +88,14 @@ function test_interface_compliance_nil {
 
 function test_import_testing_only_in_tests {
   ROOT=$( git rev-parse --show-toplevel )
-  NON_TEST_GO_FILES=$( find "${ROOT}" -iname '*.go' ! -iname '*_test.go');
+  NON_TEST_GO_FILES=$( find "${ROOT}" -iname '*.go' ! -iname '*_test.go' ! -path "${ROOT}/tests/"'*' );
```

Comment:
Please add a comment explaining the exclusion to anyone who reads this outside of this PR. I think it's easier than finding a better variable name than `NON_TEST_GO_FILES`, which now doesn't make sense.